wg-releases/wg-qa/meeting-notes/2020-02-12.md

```diff
@@ -0,0 +1,20 @@
+# QA Working Group
+
+### Date: 2020-02-12
+
+## Attendees:
+
+**Members:**
+- @cruzerld
+- @sofianguy
```

Comment:
@sofianguy i believe Anna Clegg attended this as well?

Reply:
Yes yes. My mistake. Added